src/components/ContentUploader/UploadsManager.scss

```diff
@@ -0,0 +1,118 @@
+@import '../variables';
+
+.buik.uploads-manager-container {
+    bottom: 0;
+    max-height: 0;
+    position: fixed;
+    transition: max-height .5s;
+    width: 100%;
+
+    &.is-visible {
+        max-height: 60px;
+    }
+
+    .uploads-manager-item-list {
```

Comment:
Anything specific to upload, use `bcu` as prefix. All these sub classes (is-visible, is-exapnded) should also be prefixed `bcu-` so that it will not clash with brui or other UI elements